### DISCUSSION TITLE
systemd-journal-remote for P3265-V

### DISCUSSION THREAD

**Discussion by kaleaht** [AxisCommunications/discussions#164] [General] [answered]
Hi!

I want P3265-V to send its logs to a central log server using systemd-journal-upload.service. 

Is it possible to install systemd-journal-remote to the camera?

**Comment by janssen70** [comment, marked as answer]
From firmware 10.1 there's a configurable remote syslog. see: https://help.axis.com/axis-os#syslog
I hope the link works,  otherwise search for syslog and scroll down
On older firmwares it is possible too but it takes more work. All explained in the doc
